stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ClientExtensions.kt

```diff
@@ -74,36 +66,3 @@ public inline fun <reified T : ChatEvent> ChatClient.subscribeForSingle(
 ): Disposable {
     return this.subscribeForSingle(T::class.java, listener)
 }
-
-/**
- * Runs [ChatClient.setUser] in a suspending way.
- * Throws exceptions if errors occur during the call.
- */
-public suspend fun ChatClient.setUserAndAwait(
-    user: User,
-    token: String,
-): InitConnectionListener.ConnectionData = suspendCoroutine { cont ->
-    setUser(user, token, initConnectionListener(cont))
-}
-
-/**
- * Runs [ChatClient.setUser] in a suspending way.
- * Throws exceptions if errors occur during the call.
- */
-public suspend fun ChatClient.setUserAndAwait(
-    user: User,
-    tokenProvider: TokenProvider,
-): InitConnectionListener.ConnectionData = suspendCoroutine { cont ->
-    setUser(user, tokenProvider, initConnectionListener(cont))
-}
```

Comment:
This was brand new API, and is no longer needed with `connectUser` returning a `Call`